platform-tests/src/test/java/org/eclipse/deeplearning4j/frameworkimport/tensorflow/TFGraphTestAllHelper.java

```
@@ -123,6 +123,7 @@ public ModelLoadResult apply(File file, String name) {
                 SameDiff result = tensorflowFrameworkImporter.runImport(file.getAbsolutePath(), dynamicVariables, suggestDynamicVariables);
                 return new ModelLoadResult(result, graphDef);
             }catch(Exception e) {
+                e.printStackTrace();
```

Wouldn't it be better to log this?

Or is this a debugging leftover?

---

Actually no. This actually causes a silent failure to happen if I don't do that. Printing the stack trace during the test helps uncover why a graph couldn't be imported.

---

Why not make it a loud error instead of printing to the console?

---

Done.